types/call-bind/package.json

```diff
@@ -1,7 +0,0 @@
-{
-    "private": true,
-    "types": "index",
-    "typesVersions": {
-        "<=3.9": { "*": ["ts3.9/*"] }
```

Comment:
i don’t think deleting the types entirely is the proper approach

Reply:
@ljharb For newer versions of TS (>3.9) the types are still there and still work after this change: I didn't remove [`types/call-bind/index.d.ts`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/887ff7fb0eab64c6437e88aed4f1160b5d8ffde8/types/call-bind/index.d.ts), [`types/function-bind/index.d.ts`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/887ff7fb0eab64c6437e88aed4f1160b5d8ffde8/types/function-bind/index.d.ts), etc.

For TS <=3.9 you can continue use the last version before support ended, using the e.g. ts3.9 npm tag:
`npm install call-bind@ts3.9` https://github.com/DefinitelyTyped/DefinitelyTyped/blob/09dc238010a270259288fe4441f0bd2eda7651db/README.md?plain=1#L89

This is standard when support for older versions of TS ends, I think. Here's an analogous example of removing types for TS <=3.7: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57450#issue-1067743511

Reply:
I don't care about TS 3.9; i just want to make sure these packages' types still work in modern TS. Is the package.json not required for that?

Reply:
The package.json should only be required if it specifies external dependencies afaik

Reply:
That's right. The `@types` publisher creates a package.json when one is not present in the repo.